src/containers/Tenant/Query/QueryEditor/QueryEditor.tsx

```diff
@@ -375,7 +374,7 @@ function QueryEditor(props: QueryEditorProps) {
                     <div className={b('monaco-wrapper')}>
                         <div className={b('monaco')}>
                             <MonacoEditor
-                                language={LANGUAGE_YQL_ID}
+                                language={'yql'}
```

Comment:
Can you use `LANGUAGE_ID` variable here, since you use it everywhere else?

Reply:
Unfortunately no. It seems that editor is initializing before `monaco-yql-languages` is ready. So, it's not available. And i decided to use a hardcode here, instead of another app-level constant to avoid possible clinches.